<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rps2</class>
 <widget class="QMainWindow" name="rps2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rps2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fight!</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>74</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rockButton">
       <property name="text">
        <string>Rock</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="paperButton">
       <property name="text">
        <string>Paper</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="scissorsButton">
       <property name="text">
        <string>Scissors</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose your Weapon!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>282</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRock_Paper_Scissors">
    <property name="title">
     <string>Rock Paper Scissors</string>
    </property>
   </widget>
   <addaction name="menuRock_Paper_Scissors"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
